R/estimate_infections.R

```diff
@@ -315,28 +309,31 @@ estimate_infections <- function(reported_cases,
 #' @importFrom futile.logger flog.debug
 #' @importFrom utils capture.output
 #' @inheritParams fit_model_with_nuts
+#' @inheritParams stan_opts
 #' @return A stanfit object
 #' @author Sam Abbott
 init_cumulative_fit <- function(args, samples = 50, warmup = 50,
```

Comment:
we should really depreicate this at some point. I don't think it helps and its just code to carry around